electrumsv/subscription.py

```diff
@@ -166,8 +166,15 @@ def _remove_subscription(self, entry: SubscriptionEntry, owner: SubscriptionOwne
         key = entry.key
         if key.value_type in BYTE_SUBSCRIPTION_TYPES:
             assert type(key.value) is bytes, "subscribed script hashes must be bytes"
-        subscription_id = self._subscription_ids[key]
-        subscriptions = self._subscriptions[key]
+
+        subscription_id = self._subscription_ids.get(key)
```

Comment:
This is integrating an external failure into the function, fix it externally in the source.